### PR TITLE
Added IncompleteDxfFile

### PR DESCRIPTION
**Pull request by donhfu**
Added a new key IncompleteDxfFile per request by Fukui, ADO#1282794

### CODE REVIEW

File: fatal-errors.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
     "GeodeticCoordinateSystemNotSupported": "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/48731/geoconnector#GeodeticCoordinateSystemNotSupported",
     "IModelAccessForbidden": "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/54636/knowledge-base#IModelAccessWasForbidden",
     "IModelNotFound": "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/54636/knowledge-base#iModelName",
+    "IncompleteDxfFile" : "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/54636/knowledge-base#IncompleteDxfFile",
```
Comment (jffmarker) [inline fatal-errors.json:67]: What about a name like "DxfMissingHeader"? I know it's a slippery slope, but there may be other ways a DXF is incomplete that we'll need to uniquely identify in theory.

Comment (donhfu) [reply fatal-errors.json:67]: Incomplete header will cover more cases.  If this same user adds a header but still does not have codepage set, we will run into the same problem.

Comment (jffmarker) [reply fatal-errors.json:67]: Sure... "IncompleteDxfHeader" works for me... still more future-proof than IncompleteDxfFile.

Comment (donhfu) [reply fatal-errors.json]: Renamed the key

**Review by jffmarker — APPROVED** [review]

**Pull request merged**